server/src/domain/asset/dto/asset.dto.ts

```diff
@@ -172,6 +188,20 @@ export class AssetBulkUpdateDto extends BulkIdsDto {
   @Optional()
   @IsBoolean()
   removeParent?: boolean;
+
+  @Optional()
+  @IsDateString()
+  dateTimeOriginal?: string;
+
+  @ValidateGPS()
+  @IsLatitude()
+  @IsNotEmpty()
+  latitude?: number;
+
+  @ValidateGPS()
+  @IsLongitude()
+  @IsNotEmpty()
+  longitude?: number;
```

Comment:
Those changes are not related to the PR, just a nice addition. Correct?

Reply:
It's tocheck if latitude and longitude is correct in order to modify the position of the assets

Reply:
They are related are they not? They add the option to include lat/lng/date on asset update.

Reply:
For me it's related